src/tonie_api/api.py

```diff
@@ -121,7 +121,7 @@ def get_creative_tonie(self, creative_tonie: CreativeTonie) -> CreativeTonie:
         """
         url = f"households/{creative_tonie.householdId}/creativetonies/{creative_tonie.id}"
         ct = self._get(url=url)
-        return CreativeTonie(**ct)
+        return CreativeTonie(**ct) if ct else ct

```

Comment:
I don't think returning the response is a good idea in this case. Shouldn't it be 

```python
response = self._get(url=url)
return CreativeTonie(**response) if response else None
```

The code is untested, just my understanding.